src/zope/security/tests/test_proxy.py

```diff
@@ -27,6 +27,7 @@ def coerce(*args):
         raise NotImplementedError("Not on Python 3")
     cmp = coerce
     long = int
+    unicode = str
```

Comment:
This is to satisfy linters? Good.

Reply:
Exactly. There were so many "unicode is undefined" errors it was hard to see anything else.